apps/evm/lib/evm/operation/environmental_information.ex

```diff
@@ -287,14 +287,14 @@ defmodule EVM.Operation.EnvironmentalInformation do
         machine_state: machine_state,
         exec_env: exec_env
       }) do
-    if length == 0 || length + mem_offset > EVM.max_int() do
-      0
-    else
-      wrapped_address = Helpers.wrap_address(address)
+    wrapped_address = Helpers.wrap_address(address)
 
-      account_code =
-        AccountInterface.get_account_code(exec_env.account_interface, wrapped_address)
+    account_code = AccountInterface.get_account_code(exec_env.account_interface, wrapped_address)
 
+    if length == 0 || length + mem_offset > EVM.max_int() ||
```

Comment:
This is a pretty complex `if` check. What do you think about moving it to its own line or to a private function? It may add clarity to what is going on there. What are we checking for there?

Reply:
Here we're checking memory boundaries: If we overflowed memory (`length + mem_offset > EVM.max_int()`) or if we shouldn't copy anything (`length == 0`)

Reply:
Unfortunately, it can not be moved to its own line because we're formatter :(